rust-toolchain

```diff
@@ -1 +1 @@
-nightly
\ No newline at end of file
+stable
```

Comment:
Coming back to this, I don't want to force a toolchain at all in this project. I don't think there's an advantage to this. It should realistically work on any toolchain.

Reply:
Got it -- I think the versions are installed weird on a fresh install (Gitpod / GitHub Workspaces) thus requiring nightly.